### ISSUE TITLE
Make settings editable from the admin interface

### ISSUE THREAD

**Issue by JornWildt** [cofoundry-cms/cofoundry#402]
There already exists a mechanic for adding settings using `IConfigurationSettings` (see https://www.cofoundry.org/docs/framework/configuration-settings) - but this requires someone with source code / FTP / cloud  access to update the `appsettings.json` file - and changing this file may easily break the whole site.

So I want my content editors to be able to edit some of these settings, as they are used in the site theme - but they need to be able to do so from the admin section.

One such example is the site's main title to be shown in the site header. This may need to change over time without involving any developer or administrator.

So basically I just want to implement another interface than `IConfigurationSettings` and then it shows up in the admin menu instead.

Additional question: how do I access the current `IConfigurationSettings` from the main _Layout.cshtml file (see https://stackoverflow.com/questions/66470729/how-do-i-access-cofoundry-settings-and-other-dependency-injected-classes-in-them)?

**Comment by HeyJoel** [comment]
Duplicate of #243, CMS managed settings are not properly implemented yet. A workaround is to use the custom entity system to create a "Settings" entity.

**Comment by JornWildt** [comment]
Thanks.
